### DISCUSSION TITLE
Does the release stylua-linux-aarch64.zip rely on glibc?

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [JohnnyMorganz/StyLua#741] [Q&A] [answered]
Can I use this release on the platform using musl instead of glibc?

**Comment by JohnnyMorganz** [comment, marked as answer]
Sorry, I somehow missed this when it was asked. It's been a long time, hopefully you found your answer. But if not, we do have an artifact `stylua-linux-x86_64-musl.zip` for this (introduced in #834)
